content/v1/csidriver/installation/test/certcsi.md

```diff
@@ -325,6 +315,32 @@ To run block snapshot test suite, run the command:
 cert-csi test blocksnap --sc <storageClass> --vsc <snapshotclass>
 ```
 
+#### Volume Health Metric Suite
+
+To run volume health metric test suite, run the command:
+```bash
+
+cert-csi test volumehealthmetrics --sc <storage-class> --driver-ns <driver-namespace> --podNum <number-of-pods> --volNum <number-of-volumes>
+```
+
+> Note: Make sure to enable healthMonitor for driver's controller and node pods before running this suite. It is recommended to use a smaller interval time for this sidecar.
+
+#### Ephemeral volumes suite
+
+To run ephemeral volume test suite, run the command:
+```bash
+cert-csi test ephemeral-volume --driver <driver-name> --attr ephemeral-config.properties
+--pods : Number of pods to create 
```

Comment:
Same comment as above. Write each list item in the same style/format (parallel structure)